index.html

```diff
@@ -1,39 +1,48 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="stylesheet" href="style.css">
   <title>Document</title>
 </head>
+
 <body>
   <h1>Awesome books!</h1>
 
-  <table>
-    <thead>
-      <tr>
-        <td><strong>Title</strong></td>
-        <td><strong>Author</strong></td>
-      </tr>
-    </thead>
-    <tbody id="books-list">
-      
-    </tbody>
-  </table>
+  <div class="container table-wrapper">
+    <table>
+      <thead>
+        <tr>
+          <td><strong>Title</strong></td>
+          <td><strong>Author</strong></td>
+        </tr>
+      </thead>
+      <tbody id="books-list">
```

Comment:
- [ ] For the book listing, You will notice that each book title is joined to the book author using the `by` prefix following the wireframe given.